engineering/licensing.md

```diff
@@ -0,0 +1,37 @@
+# Licensing Policy
+
+This document describes the source{d} licensing policy for the open-source
```

Comment:
Please, honour https://github.com/src-d/guide/issues/39, no one has disagreed with it so far.